scripts/build.ps1

```diff
@@ -1,100 +0,0 @@
-﻿Param(
-    [string]$Configuration="Debug",
-    [string]$Restore="true",
-    [string]$Version="<default>",
-    [string]$BuildVersion=[System.DateTime]::Now.ToString('preview2-yyMMdd-1'),
-    [string]$SkipTests="false"
-)
-
-Write-Host "Configuration=$Configuration."
-Write-Host "Restore=$Restore."
-Write-Host "Version=$Version."
-Write-Host "BuildVersion=$BuildVersion."
-Write-Host "SkipTests=$SkipTests."
-
-$dotnetExePath="$PSScriptRoot\..\dotnetcli\dotnet.exe"
-if ($Version -eq "<default>") {
-    $Version = (Get-Content "$PSScriptRoot\..\DotnetCLIVersion.txt" -Raw).Trim()
-}
-
-if (!(Test-Path "$PSScriptRoot\..\dotnetcli")) {
```

Comment:
With us reworking the corefxlab build scripts/infra, can you please explain how contributors build/test corefxlab now, what dependencies (if any) are needed, and more importantly how to use the packages?

We should update the README accordingly, specifically these sections:
https://github.com/dotnet/corefxlab#how-to-use - This still mentions the use of the MyGet feed
https://github.com/dotnet/corefxlab#building-and-testing - This currently explains where the CLI is downloaded, how to set the path, and how the tests for the whole corefxlab.sln work.

Reply:
+1. You should be able to mostly copy paste from the rest of our repos’ Readmes since we are now going to have very similar instructions to the basic repos